cli_app/pipeline.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+
+class Processor:
+    def __init__(self):
```

Comment:
constructor must initialize all required variable, please complete contructor first, and include important parameters which will be passed when called contsructor:
path, file_format and name_format